acceptance/apps/rails_test.go

```diff
@@ -89,29 +92,37 @@ var _ = Describe("RubyOnRails", func() {
 		out, err = env.Epinio("", "apps", "env", "set", rails.Name, "RAILS_MASTER_KEY", rails.MasterKey)
 		Expect(err).ToNot(HaveOccurred(), out)
 
-		// Create a database for Rails
-		out, err = proc.Run("", false, "helm", "repo", "add", "bitnami", "https://charts.bitnami.com/bitnami")
+		// Create a custom service catalog
+		serviceName = names.Truncate(catalog.NewServiceName(), 20)
+		catalogName = names.Truncate(catalog.NewCatalogServiceName(), 20)
+
+		out, err = proc.RunW("sed", "-i", "-e", fmt.Sprintf("s/myname/%s/", catalogName), testenv.TestAssetPath("my-postgresql-custom-svc.yaml"))
```

Comment:
This will work for as long as no other test tries to `sed` the same file. Generally, when a test needs a file we make it a temp one with a random name. Sorry for the post-merge comment.